business_requirement_deliverable/models/business.py

```diff
@@ -12,6 +12,15 @@ class BusinessRequirementResource(models.Model):
     _description = "Business Requirement Resource"
 
     sequence = fields.Integer('Sequence')
+    state = fields.Selection(
+        related='business_requirement_id.state',
+        selection=[('draft', 'Draft'), ('confirmed', 'Confirmed'),
```

Comment:
@sudhir-serpentcs Please follow style: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms

>When making a comma separated list, dict, tuple, ... with one element per line, append a comma to the last element. This makes it so the next element added only changes one line in the changeset instead of changing the last element to simply add a comma.

```
partner_id = fields.Many2one(
    "res.partner",
    "Partner",
    "Required",
)
```